app/models/event_role.rb

```diff
@@ -112,12 +112,16 @@ def to_s
     role + ": " + assigned_to
   end
   
-  def assigned_to(options = {})
+  def assigned_to(mode = :full_name)
+    # Logger
     if assigned?
-      if options[:use_display_name]
```

Comment:
We still need to honor `:use_display_name` because that determines when a display name should never be shown (i.e. on the home page when not logged in).

Reply:
I still honoured it, just refactored the function to not receive an options parameter. The default is still to show the "full" name, and only showing the display name if it is specifically requested.